toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -191,14 +192,14 @@ func getBuildDependencies(node *pkggraph.PkgNode, pkgGraph *pkggraph.PkgGraph, g
 
 // parseCheckSection reads the package build log file to determine if the %check section passed or not
 func parseCheckSection(logFile string) (err error) {
-	file, err := os.Open(logFile)
+	logFileObject, err := os.Open(logFile)
 	// If we can't open the log file, that's a build error.
 	if err != nil {
 		logger.Log.Errorf("Failed to open log file '%s' while checking package test results. Error: %v", logFile, err)
 		return
 	}
-	defer file.Close()
-	for scanner := bufio.NewScanner(file); scanner.Scan(); {
+	defer logFileObject.Close()
+	for scanner := bufio.NewScanner(logFileObject); scanner.Scan(); {
```

Comment:
Is the logfile re-written on the next loop?  It seems to me that the logfileObject will not have been closed, or flushed to disk.  I am worried this is going to either append to the logfileObject from the previous loop or there's an error here because the logfile is already open.

Reply:
`logFileObject` isn't written to at any point during this function; the loop just feeds the log file one line at a time to make parsing cleaner.